Update credentials setup message

vianasw opened the pull request:
We're updating the message shown when credentials are not set up in Calypso blue with the same with we have in Calypso green, for feature parity and to avoid customers contacting support because they think credentials are needed for backups to work.

<!--- Provide a general summary of your changes in the Title above -->
<!-- Would you like this feature to be tested by Beta testers?
Please add testing instructions to projects/plugins/jetpack/to-test.md in a new commit as part of your PR. -->
<!-- a12s: If you have an expected version that you're aiming for the PR to add, please use the Milestone field to communicate it. If you leave it blank, that indicates there isn't a preference. -->

Fixes #

#### Changes proposed in this Pull Request:
<!--- Explain what functional changes your PR includes -->
* We changed the title and message in the settings card to clarify that server credentials are needed for restores. 

#### Other information:

- [ ] Have you written new tests for your changes, if applicable?
- [ ] Have you checked the E2E test CI results, and verified that your changes do not break them?

#### Jetpack product discussion
p7pQDF-7co-p2#comment-21470

#### Does this pull request change what data or activity we track or use?
<!--- If so, please add the "[Status] Needs Privacy Updates" label and explain what changes there are. -->
<!--- Check existing Jetpack support documents for a preview of the information we need. -->
No

#### Testing instructions:
<!-- If you were reviewing this PR, how would you like the instructions to be presented? -->
<!-- Please include detailed testing steps, explaining how to test your change. -->
<!-- Bear in mind that context you working on is not obvious for everyone.  -->
<!-- Adding "simple" configuration steps will help reviewers to get to your PR as quickly as possible. -->
<!-- "Before / After" screenshots can also be very helpful when the change is visual. -->

* Purchase a Jetpack Backup plan
* Go to the Jetpack dashboard "At a glance" section
* Check that the text Backup card under Security section is: "Enter your SSH, SFTP or FTP credentials to enable one-click site restores and faster backups"
* Go to the Security settings page
* Check that the text is: "Add SSH, SFTP, or FTP credentials to enable one-click site restores"
* Purchase a Jetpack Scan plan
* Go to the Jetpack dashboard "At a glance" section
* Check that the text is: "Add SSH, SFTP, or FTP credentials to enable one-click fixes"
* Go to the Security settings page
* Check that the text under the Security section is: "Add SSH, SFTP, or FTP credentials to enable one-click site restores and fixes"

vianasw committed:
Update credentials setup message

We're updating the message shown when credentials are not set up in
Calypso blue with the same with we have in Calypso green, for feature
parity and to avoid customers contacting support because they think
credentials are needed for backups to work.

vianasw committed:
Update credentials message for consistenty

Updating the message around server credentials in two more places for
consistency.

vianasw committed:
Fix typo

vianasw committed:
Update credentials message for the Scan section

Yet another place that needed a copy change for consistency reasons.
Credentials are needed to enable one click fixes for Scan.

vianasw committed:
Replace one click with one-click in credentials texts

We're already using the hyphenated word in support documentations and
other places.

vianasw committed:
Fix linting error

nunyvega reviewed (dismissed):
Reviewed and tested, it works as expected :)

One minor note, not a blocker: in the Jetpack Dashboard, I would add a full stop to the two phrases for consistency with all the rest of the text.

vianasw committed:
Update wording on credentials setup messages

We're trying to improve clarity and increase consistency with other
places where we show a credentials setup banner.

Commit:
Merge branch 'trunk' into update/credentials-setup-message

nunyvega reviewed (approved):
The changes look good. It also tests well!

🚢
